### PR TITLE
Tasks 935-937: Defeasible reasoning task/dataset

### PR DESCRIPTION
**Pull request by Sujan242**
Tasks based on: https://aclanthology.org/2020.findings-emnlp.418.pdf
Suggested in  issue #119

### CODE REVIEW

File: tasks/README.md [hunk 1]
```diff
@@ -480,6 +480,23 @@ Name | Summary | Category
...
+`task610_conllpp_ner` | Recognize and label proper nouns (Named Entity Recognition) | NER/Labeling  
+`task563_discofuse_answer_generation` | Answer Generation on Dataset Card for DISCOFUSE | Answer Generation
+`task550_discofuse_sentence_generation` | Senetence Generation on Dataset Card for DISCOFUSE | Senetence Generation
+`task564_discofuse_classification` | Classification on Dataset Card for DISCOFUSE | Classification
+`task562_alt_language_identification` | Language Identification on Dataset Card for ALT | Language Identification
+`task935_defeasible_nli_atomic_classification` | Given a premise, hypothesis and an update, indentify whether the update strengthens or weakens the hypothesis. | Classification
+`task936_defeasible_nli_snli_classification` | Given a premise, hypothesis and an update, indentify whether the update strengthens or weakens the hypothesis. | Classification
+`task937_defeasible_nli_social_classification` | Given a hypothesis and an update, indentify whether the update strengthens or weakens the hypothesis. | Classification
```
Comment (danyaljj) [inline tasks/README.md]: I think merge was not done properly that we have a couple of green lines here.

**Comment by danyaljj** [comment]
Thanks for the PR!

@tanay2001  wondering if you can review this PR?

**Comment by tanay2001** [comment]
Hi,

For the definition would it be better to rephrase it like:
 `In this task, you are given a premise, a hypothesis, and an update. Your task is to output 'strengthener' or 'weakener' if the update strengths or weakens the hypothesis respectively .`

**Comment by danyaljj** [comment]
@Sujan242, thanks for the PR! 

I think we might be able to improve the definition, by improving how we define each field. 
See page 14 of the relevant paper, which has the definitions used by the authors: https://aclanthology.org/2020.findings-emnlp.418.pdf

**Comment by Sujan242** [comment]
Thanks for the review! 
I have updated the definitions for all three tasks.

**Comment by danyaljj** [comment]
Thanks! 

 - `true.The`: missing space after the punctuation, in several places.

**Comment by Sujan242** [comment]
> Thanks!
> 
> * `true.The`: missing space after the punctuation, in several places.

I have made these changes

**Comment by danyaljj** [comment]
Looks good to me! Leaving it here for @swarooprm to give it a 2nd look before merging it.

**Comment by danyaljj** [comment]
@Palipoor @aarunku5 one you could give it a 2nd look? If it looks good to you, feel free to merge it.

**Comment by Palipoor** [comment]
Looks good to me!

**Review by Palipoor — APPROVED** [review]

**Pull request merged**